examples/resource-runtime-launch.cpp

```diff
@@ -153,7 +153,7 @@ int main(int argc, char *argv[])
 #endif
 
   //Get typed erased resource - it will internally store if we are running on the host or device
-#if defined(RAJA_ENABLE_CUDA) || defined(RAJA_ENABLE_HIP)
+#if defined(RAJA_GPU_ACTIVE) && !defined(RAJA_ENABLE_SYCL)
```

Comment:
Does the runtime selection not make sense for SYCL or is it not working?

Reply:
This particular example uses reductions which are not yet supported in SYCL. The dynamic matrix-transpose example however does illustrate run-time selection with SYCL.